jobs/correction-ben-statement/add_registrars_notation.ipynb

```diff
@@ -66,18 +108,50 @@
    "outputs": [],
    "source": [
     "from urllib.parse import urljoin\n",
-    "from rn_output import rn_businesses\n",
+    "from rn_output import businesses\n",
     "\n",
     "current_date = datetime.now().date().isoformat()\n",
     "headers = {\n",
     "    'Content-Type': 'application/json',\n",
     "    'Authorization': 'Bearer ' + token\n",
     "}\n",
     "\n",
+    "successful_identifiers = []\n",
     "failed_identifiers = []\n",
+    "skipped_identifiers = []\n",
     "\n",
     "# loop through list of businesses to create filing\n",
-    "for business in rn_businesses:\n",
+    "for identifier in businesses:\n",
+    "    filing_details = %sql \\\n",
+    "        SELECT f.id, f.filing_date \\\n",
+    "        FROM businesses b \\\n",
+    "        JOIN filings f ON b.id = f.business_id \\\n",
+    "        WHERE f.filing_type = 'incorporationApplication' \\\n",
+    "        AND b.identifier = :identifier\n",
```

Comment:
Keep a note to replace this query with alteration while running against altered BEN businesses (or create a new `add_registrars_notation_alteration.ipynb` file specific to alteration)